.travis.yml

```diff
@@ -1,50 +1,37 @@
-language: python
+language: node_js
+node_js: 6
 sudo: required
+services: docker
 dist: trusty
-python: '3.4'
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
 env:
   global:
-    - EQ_ENABLE_CACHE=True
-    # Codacy
-    - secure: q3hyZhRj24C5sT7T0ViTDCwhT8eVjpanTf6BkPPBI43/6NBJeF1vi4/LkfJMQa0qwYibu5zB8jtkk2VsqGRMGvj9fMufOKXGqebOsPYWUJl1MZM3t7OZS57xF9wBUenZwx1oyb/VPrFB56ctGny1XEbF4l47NmbKWD1iMWG1f3Y6PCJ7o8o62v0irmfbdYFTsWjipCQlCrKZm+KVWZD25egIDJMDpq9e0nYNoSW+5fMXDGOtV20/U/rcmW5Lfn06ppJ0C4CaKHuM+MzZtZJwtiCI2TqF0ihQl7ry7YgpqEZLPHQYQ5npIpecxP1f/6ebcAPdKKkJ2Ccutrv/7IFlJo7rWLTB7mzJ4ep11HCdo3mL+JREp1HUJLeEngXFC3lKRXPHKukUxe4Lkz1/g6kdwCUZxajGTwML84FHmqoZxKGMKhMN+0n6rO8wEUR32q4+JbO5z/oQGhtjQZkogq8vWY6+/aq328hsbmy72s23AiX+BQUHqk2QFs3T9FQiqwn4OqqX4JKzmHs838LT+dUTFgH+mWo0lalxa3Pr7cJAwhn97VSnIdU9JTC8RyPWbAHkah78PXlL9guHI1nzQTjTcMm/yTlmtE2NxTSY+ZGIV+O5pcJm3HkAjZvNi/0jjDWwBHs/rZgj1t7Eoueh8dPTSZ12LmqKuBpvkrzLKcs2zt4=
+    - EQ_FUNCTIONAL_TEST_ENV=docker
   matrix:
-    - EQ_FUNCTIONAL_TEST_SUITES="core,ukis"
-    - EQ_FUNCTIONAL_TEST_SUITES=census
+    fast_finish: true
+    include:
+      - EQ_FUNCTIONAL_TEST_SUITES="core,ukis"
+      - EQ_FUNCTIONAL_TEST_SUITES=census
 before_install:
-    # install Chromedriver
-    - export CHROME_BIN=/usr/bin/google-chrome
     - export DISPLAY=:99.0
     - sh -e /etc/init.d/xvfb start
-    - wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-    - sudo dpkg -i google-chrome*.deb
-
-    - sudo apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg
-    - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-    - sudo apt-get update -qq
-    - sudo apt-get install -q libappindicator1 fonts-liberation yarn
-
-    # Repo for newer Node.js versions
-    - curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 cache:
-    - pip
     - yarn
-install:
-    - . $HOME/.nvm/nvm.sh
-    - nvm install v6.9.1
-    - pip install -U pip wheel
-    - pip install --require-hashes -r requirements.txt
-    - pip install -e git+https://github.com/ONSdigital/cryptography.git@4a90c254278231d7defeac304a3cfd752e96e786#egg=cryptography
-    - pip install -r requirements_for_test.txt
-
 before_script:
-    - yarn compile
-    - ./scripts/run_app.sh
+    - docker-compose --version
+    - docker-compose -d up
+    - docker --version
+    - docker tag eqsurveyrunner_eq-survey-runner onsdigital/eq-survey-runner
+    - docker build -t onsdigital/eq-survey-runner-unit-tests -f Dockerfile.test .
+    - docker run onsdigital/eq-survey-runner-unit-tests
+    - yarn
```

Comment:
It's needed to run the Selenium tests; did discuss with Hamish about splitting the testing part out from the app part, they're currently all one thing.